.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  test-evaluation-js:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: 'evaluation/js'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/Iron'
+          cache: 'npm'
+          cache-dependency-path: '**/**/package-lock.json'
+      - run: npm ci
+      - run: npm run build
+      - run: npm test
```

Comment:
Note: currently not tests exist for the `evaluation/js` directory, but I put this in thinking maybe it would make sense to add them? Happy to remove if it does not make sense to add tests there.